libvast/builtins/endpoints/query.cpp

```diff
@@ -468,25 +588,37 @@ request_multiplexer_actor::behavior_type request_multiplexer(
         } else {
           return rq.response->abort(422, "missing parameter 'query'\n");
         }
-        auto parse_result = system::parse_query(*query_string);
+        auto parse_result = pipeline::parse(*query_string);
```

Comment:
I think this actually needs a changelog entry, as we now no longer allow a leading expression before the pipeline in the /query endpoint.